TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -42,13 +42,13 @@ public class Workspace
         private CopyWatcher _CopyWatcher;
         private System.Timers.Timer _semanticUpdaterTimer;
         private bool _timerDisabled;
-
+        private readonly Dictionary<Uri, DocumentContext> _openedDocuments;
```

Comment:
We should probably start using _Concurrent​Dictionary<Uri, DocumentContext>_ this will avoid to use a lock variable.

Reply:
I've considered using it at first, but it does not fit for all usages in the class. Sometimes it translates well (simple access can be replaced with corresponding method in `ConcurrentDictionary`), but for more complex operations like refreshing all opened documents, it is not well suited. Locking would still be necessary and so I prefer to leave as it is.